charts/tidb-operator/templates/advanced-statefulset-deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: advanced-statefulset-controller
+  namespace: pingcap
+  labels:
+    app: advanced-statefulset-controller
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: advanced-statefulset-controller
+  template:
+    metadata:
+      labels:
+        app: advanced-statefulset-controller
+    spec:
+      containers:
+      - name: advanced-statefulset-controller
+        image: quay.io/cofyc/advanced-statefulset:latest
```

Comment:
Can we put this under pingcap now? Including the github repo?

Reply:
yes, when this solution works well